### DISCUSSION TITLE
O que vocês acham de faculdade de T.I? vale a pena?

### DISCUSSION THREAD

**Discussion by Ryamsousa16** [frontendbr/forum#1920] [Discussão]
Eu estava vendo muitas pessoas próximas criticando as faculdades de T.I, então eu resolvi abrir essa discussão. O que vocês acham? E se possível diga o que você achou da faculdade se você tiver feito, me ajudaria muito para tomar algumas decisões.

**Comment by carlosviniciusananias** [comment]
> Eu estava vendo muitas pessoas próximas criticando as faculdades de T.I, então eu resolvi abrir essa discussão. O que vocês acham? E se possível diga o que você achou da faculdade se você tiver feito, me ajudaria muito para tomar algumas decisões.

Cara, sou formado em **ads** há 02 anos e hoje sou desenvolvedor atuando com comércio digital.

A grade curricular da maioria das instituições não é "atualizada" com o mercado e muita coisa que você aprende lá não é utilizada em nosso cotidiano.
A faculdade te proporciona os primeiros passos, a base (por exemplo a materia de algoritmos -- super importante), dai vai da sua curiosidade buscar mais, querer aprender mais e se aprofundar em uma área específica.
O que pude perceber é que fui realmente "aprender" somente quando entrei no mercado!

Na minha opinião acho importante a base de conhecimentos que o curso me ofereceu e também sinto que o diploma ainda pesa em alguns casos... (Mas diploma não é conhecimento, é claro!)

Abraços,

**Comment by expalmer** [comment]
Aprender é bom, faculdade é bom, presencial melhor ainda pois conhece gente, troca idéia, faz network. Faculdade é perfeita ? Não.
E diploma em instituição confiável pesa, pois significa que vc passou por um processo, estudou, aprendeu conceitos.

**Comment by ninetails** [comment]
Tem thread pra caramba sobre o assunto, aconselho dar uma olhada no histórico :D

Eu acho fazer faculdade situacional.

É um diferencial quando você tiver numa concorrência em processo seletivo, no caso de dois perfis semelhantes. Se você não tem diferencial em experiência profissional, a academia vai ser um.

Também lembro de já terem citado previamente no fórum que alguns países dão preferência para graduados, caso você algum dia pense ir para fora.

Também há a possibilidade de seguir carreira acadêmica, caso for algo que tenha interesse.

Eu fiz duas, não terminei ambas. Aprendi muito com elas e eu ressalto isso: você não necessariamente vai aprender coisas que você vá utilizar no trabalho, mas vai te dar base para muita coisa que você vai fatalmente ver no trabalho, como Estrutura de Dados e Algoritmos. Também tem muita faculdade que a galera critica por ter grade defasada, pra mim não é um argumento válido pois Estruturas de Dados e Algoritmos claro que evoluem, mas saber o básico já ajuda profissionalmente qualquer um que almeja programação.

O mercado para nossa área está bem aquecido em relação a oportunidades, e existem para todos os perfis sejam graduados ou não graduados.

O que eu acho é que cada um tem suas metas e o que quer da carreira. Fazer ou não é uma opção pessoal, dado o que a pessoa quer para si mesma profissionalmente.

**Comment by renedocket** [comment]
Eu acredito que tudo depende, a minha experiência na faculdade foi muito mais voltada pra conhecer pessoas e entender visões diferentes do que de fato aprender. Como moro no interior e o investimento em tech aqui é quase zero, tive que fazer mesmo assim, por que ir pra SP capital não tinha como, pois eu trabalhava aqui. Então escolhi continuar mesmo assim e estudando por fora como outras pessoas comentaram acima. Além de pessoas que me ajudaram me dando toques do que estudar pra conseguir uma vaga e tals, e que realmente o que a gente aprende não está de acordo com o mercado.

Hoje eu trabalho como desenvolvedor front-end em SP já faz 2 anos, pois aqui onde moro não tem emprego de tech como em Sp, e acredito que o que a faculdade me ensinou foi, não dependa apenas da faculdade pra conseguir entrar na área. 

Estude, se esforce, faça projetos por fora! Recomendo a faculdade pelo fato de ela abrir sua visão pra muitas coisas te dando um leque de possibilidades, além de conhecer pessoas! Eu só conseguir de fato entender como funciona a área quando eu entrei e coloquei a mão na massa, porém, conceitos vistos na faculdade acabam aparecendo no dia a dia. 
Na minha cabeça não é regra fazer facul, também fiquei nesse mesmo dilema na época, como eu não tinha muita opção, fiz por aqui. Mas se você tem a opção de fazer um facul que é diretamente ligada a tech e mais atual, eu recomendo.

Mas como eu disse, tudo depende.

**Comment by jonathannto** [comment]
Aqui em Brasília ter faculdade é super importante a maioria das empresas atendem a contratos do governo e em quase sua totalidade eles exigem graduação e tempo de experiência, logo a maioria das vagas para trabalho pedem graduação.

**Comment by peidrao** [comment]
Antes achava fundamental para arrumar emprego, hoje, na metade do curso de tecnologia da Informação (UFRN) não vejo como tão importante. Pois, meus objetivos estão bem distantes da grade curricular (gosto mais da parte web). Entretanto, acredito que seja muito importante ter a experiência de cursar uma faculdade, pois existem muitos fundamentos que você não vai encontrar em cursos que hoje a internet possuem a rodo. A experiência conta muito, além das amizades e contatos que são possíveis fazer durante o curso. Outro fator importante é a questão de procurar vagas, as empresas priorizam (infelizmente)  muito pessoas que possuem alguma formação. Claro que, existem pessoas que são incrivelmente dotadas de inteligência e conseguem emprego sem nenhuma graduação e muito menos técnico.

**Comment by lucasferreiralimax** [comment]
Eu tenho muito interesse em me graduar e melhorar meus conhecimentos teóricos sobre tudo que é voltado a ciência da computação e engenharia de software, acho bastante valido buscarmos certificados além de implementar os conhecimentos em projetos científicos validos para a sociedade.

Ainda não consegui entrar em nenhuma universidade pública aqui no Brasil, mas mantenho meus estudos constantes e o desenvolvimento para conseguir evoluir e ingressar em uma algum um dia.

Mesmo não estando em uma universidade e não tendo acesso ainda, não devemos parar de estudar, sonhar e melhorar nossas habilidades seja técnica, teórica ou pratica.

Quem dita nosso futuro? Para muitos pode ser perca de tempo estudar tanto e buscar melhorar progressivamente, mas para poucos pode ser uma forma de melhorar a vida e evoluir para ajudar muitas pessoas com aquilo que conhece e desenvolve.

Não tenho faculdade ainda mas é um dos meus sonhos a ser realizado ter uma graduação para dar continuidade ainda mais na carreira acadêmica e melhorar com tudo que posso aprender nesse percurso. 

**Vale a pena todo o conhecimento que você luta para aprender**

**Comment by willianjusten** [comment]
Acho que todo mundo já respondeu muito do que eu acho e concordo com todos os pontos. Eu tenho um post onde falo um pouco também, caso você se interesse:

https://willianjusten.com.br/devo-fazer-faculdade/

**Comment by lucasrmagalhaes** [comment]
Acho válido mas precisa estudar por fora para encarar o mercado. Mudei de curso de Sistemas para Internet para ADS porque acho mais completo.
